content/en/blog/_posts/Securing-Admission-Controllers.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: blog
+title: "Securing Admission Controllers"
+date: YYYY-MM-DD
+slug: text-for-URL-link-here-no-spaces
+---
+
+# Securing Admission Controllers
```

Comment:
This is the title, you should omit it from the Markdown (it will be rendered in from the front matter).

Reply:
removed.